### ISSUE TITLE
Multithreading in Java

### ISSUE THREAD

**Issue by Edylene-Nyairaw** [section-engineering-education/engineering-education#3761]
### Proposed title of article
Multithreading in Java

### Introduction paragraph (2-3 paragraphs):
As Java is a multi-threaded programming language, we can use it to create multi-threaded programs. A multi-threaded program is made up of two or more portions that can operate in parallel, each of which can tackle a distinct task at the same time, maximizing the use of available resources, especially when your computer has several CPUs.

Multitasking is defined as the sharing of common processing resources, such as a CPU, by many processes. Multi-threading extends the concept of multitasking into programs by allowing you to divide certain operations within a single app into several threads. Each thread can execute in its own thread.Not only does the operating system split processing time among distinct apps, but it also divides it among each thread within an application.

Multi-threading allows you to write a software in which multiple activities can run at the same time.

### Key takeaways: 
By the end of the tutorial, the leaner will understand;
-Life Cycle of a Thread
-how to create a Thread by Implementing a Runnable Interface
-how to Create a Thread by Extending a Thread Class
-how to Running Threads
-how to Thread Methods
-how to deal with Concurrency Problems
-Major Java Multithreading Concepts
### References: 
N/A

**Comment by WanjaMIKE** [comment]
Good afternoon and thank you for submitting your topic suggestion. @Edylene-Nyairaw 
Your topic form has been entered into our queue and should be reviewed (for approval) as soon as a content moderator is finished reviewing the ones in the queue before it.

**Comment by hectorkambow** [comment]
@Edylene-Nyairaw 
How will this article differ from what is [published our other blog sites](https://www.google.com/search?q=Multithreading+in+Java&rlz=1C1CHBF_enUS891US891&ei=HOtAYdfbF9H--wTk84uABA&oq=Multithreading+in+Java&gs_lcp=Cgdnd3Mtd2l6EAMyCAgAEIAEELEDMgUIABCABDIFCAAQgAQyBQgAEIAEMgUIABCABDIFCAAQgAQyBQgAEIAEMgUIABCABDIFCAAQgAQyBQgAEIAEOgcIABBHELADSgQIQRgAUK6DCliugwpg64QKaAJwAngAgAFeiAFekgEBMZgBAKABAqABAcgBCMABAQ&sclient=gws-wiz&ved=0ahUKEwjX2Yecjf_yAhVR_54KHeT5AkAQ4dUDCA4&uact=5)

**Comment by WanjaMIKE** [comment]
Unfortunately, multithreading in Java is already covered. Thanks https://github.com/section-engineering-education/engineering-education/pull/2432
